Library/Homebrew/test/testing_env.rb

```diff
@@ -1,6 +1,18 @@
 $:.unshift File.expand_path("../..", __FILE__)
 $:.unshift File.expand_path("../lib", __FILE__)
 
+# This must be at the top
+if ENV["HOMEBREW_TESTS_COVERAGE"]
+  require "simplecov"
+  SimpleCov.start do
+    SimpleCov.root File.expand_path("#{File.dirname(__FILE__)}/..")
+    add_filter do |src|
+      src.filename.include?("Homebrew/test") ||
+      src.filename.include?("vendor/bundle")
+    end
+  end
```

Comment:
We’re at 59.59%. Let’s use 50 then we’ll bump it to 60 when we’ll have more tests.